docs/AccessScalarProducts.md

```diff
@@ -0,0 +1,185 @@
+# Access Scalar products on a Kubernetes environment
+
+This document explains how to access Scalar products on a Kubernetes environment. On a Kubernetes environment, you can access Scalar products via Scalar Envoy. To access this Scalar Envoy via service resource of Kubernetes named as `<helm release name>-envoy`. There are several ways to access this `<helm release name>-envoy` service resource as follows.
```

Comment:
I think `access` might be too blurry here.
We probably mean requesting transactions to Scalar products, right?
I first thought it was for doing `kubetl get pods` to Scalar products.

Reply:
@feeblefakie  
Thank you for your review!

> I think `access` might be too blurry here.
> We probably mean requesting transactions to Scalar products, right?

Yes. That's right.
I agree `access` is blurry.

I think there are several options instead of `access Scalar product` as follows.

* run client requests to Scalar products
* execute client requests to Scalar products
* accept client requests for Scalar products
* listen for client requests on Scalar products
* expose Scalar products to run/execute client requests

And, in my opinion, I think we can update each title/section title as follows. (Also, I will update all statements in this document based on each title.)

* h1 (title): Expose Scalar products on a Kubernetes environment
* h2 (section title): Run client requests to Scalar products via service resources directly from inside of the Kubernetes cluster
  * Note: In this case, we don't need additional operations other than deploying to expose Scalar products. Scalar Envoy can accept client requests from inside the Kubernetes cluster by default.
* h2 (section title): Expose Scalar products via Load Balancers to run client requests from outside of the Kubernetes cluster
* h2 (section title):  Run client requests to Scalar products from a bastion server (For testing purposes only / Not recommended in the production environment)

The most popular use case in the production environment is `Expose Scalar products via Load Balancers`, so I think it is better to use `expose` for the document title, too.

What do you think?